scripts/linux/install-daemon-upstart.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+SCRIPT_TITLE="install-daemon"
+PKG_INSTALLER="apt-get" # supports : sudo ${PKG_INSTALLER} install <pkgname> 
+
+CHK_PKGS=( "bzip2" "tar" )
+RDB_DAEMON="ravendbd"
+
+NC='\033[0m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+PURPLE='\033[0;35m'
+CYAN='\033[0;36m'
+BLUE='\033[0;34m'
+
+REPORT_FILE='/tmp/install-daemon.ravendb.report.log'
+OUT_FILE='/tmp/install-daemon.ravendb.out.log'
+
+RAVEN_UBUNTU_PKG="../../artifacts/ubuntu.`lsb_release -r | awk '{print $2}'`-x64"
+RDB_WATCHDOG="ravendb.watchdog.sh"
+
+if [ "$1" == "" ]
+then
+	RAVEN_PARAMS="--Raven/ServerUrl=http://localhost:8080 --Raven/LogsDirectory=RDB_RAVENDB_PATH/../Logs --Raven/DataDir=RDB_RAVENDB_PATH/../Databases --Raven/StudioDirectory=RDB_RAVENDB_PATH/Server"
```

Comment:
Shouldn't we just follow what's in `settings.json` instead of overriding `ServerUrl` and other stuff?

Going further (not for this time around I think) we probably should think of lay out stuff in Linux as people are used to - you know "when in Rome.." :) I mean using places like `/etc/ravendb/settings.json` and `/var/lib/ravendb` for data etc.

Reply:
Ideally, I want to have just one place for these settings, and `settings.json` is the correct one, I think

Reply:
https://github.com/ravendb/ravendb/pull/8885/commits/57495de8703a719eac044db2deea831ef96ef552